raiden/tests/integration/api/test_restapi.py

```diff
@@ -1353,8 +1353,7 @@ def test_api_payments_with_hash_no_secret(
         ),
         json={"amount": amount, "identifier": identifier, "secret_hash": secret_hash},
     )
-    with watch_for_unlock_failures(*raiden_network):
-        response = request.send().response
+    response = request.send().response
```

Comment:
This an two other tests look to me like unlock failures are actually expected, and the old `watch_for_unlock_failures` did not catch them for some reason. They failed in the last build before `watch_for_unlock_failures` was removed.